lib/screens/home/home_screen.dart

```diff
@@ -22,13 +22,24 @@ class _HomeScreenState extends State<HomeScreen> {
   CalendarController _calendarController = CalendarController();
   final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
   Stream<List<Wod>> _wodsStream;
+  Map<DateTime, List<Wod>> _data;
 
   @override
   void initState() {
     super.initState();
     _wodsStream = context.read<WodRepository>().getWods();
   }
 
+  _groupWods(List<Wod> wods) {
+    _data = {};
+    wods.forEach((wod) {
+      DateTime date =
+          DateTime.utc(wod.date.year, wod.date.month, wod.date.day, 12);
+      if (_data[date] == null) _data[date] = [];
+      _data[date].add(wod);
+    });
+  }
```

Comment:
When you change that thingy of fetching just the workouts of that day from Firestore, you won't need need this function.

Nevertheless, I'm happy you could do this programming logic. Well done 😄